spec/shr_oncology.txt

```diff
@@ -82,31 +85,295 @@ includes 0..1	BreastCancerHistologicGrade
+
 		
-		Element:		BreastCancerPrimaryTumorClassification
-		Concept:		SCT#78873005 "T category"
-		Based on:		CodedEvaluationComponent
-		Description:	"The size and extent of the primary tumor, based on criteria defined by the staging system being used."
-		Value:			CodeableConcept
-		0..1			StageSuffix
+		EntryElement:	TumorAssessment		// change to TumorAssessment
+		Based on:		LaboratoryObservation
+		Concept:		SCT#395531003 "Tumor observable (observable entity)"
+		Description:	"An abnormal mass of tissue that results when cells divide more than they should or do not die when they should."  // source: NCI 
+		0..0			ObservationContext.ResultValue
+		0..0			ObservationContext.ExceptionValue
+						LaboratoryObservationTopic.TopicCode is SCT#395531003 "Tumor observable (observable entity)"
+		//				LaboratoryObservationTopic.Specimen is type BreastSpecimen
+		0..0			LaboratoryObservationTopic.Specimen
+		0..0			LaboratoryObservationTopic.Precondition
+		0..0			LaboratoryObservationTopic.FindingMethod
+		0..0			LaboratoryObservationTopic.Media
+		0..0 			LaboratoryObservationTopic.EvaluationComponent
+						LaboratoryObservationTopic.PanelMembers.Observation
+		includes 0..1	TumorDimensions
+		includes 0..1	TumorMargins
+
+			Element: 	TumorDimensions
+			Based on:		LaboratoryObservation
+			Concept:		SCT#399375005 "Tumor size (observable entity)"
+			Description:	"Dimension size of the tumor sample evaluated for staging.  Dimensions include a height, width, and margin measurements."
+			0..0			ObservationContext.ResultValue
+			0..0			ObservationContext.ExceptionValue
+							LaboratoryObservationTopic.TopicCode is SCT#263605001 "Tumor size finding (finding)"
+							LaboratoryObservationTopic.DiagnosticService is #pathology
+			0..0			LaboratoryObservationTopic.FindingMethod
+			0..0			LaboratoryObservationTopic.Precondition
+			0..0			LaboratoryObservationTopic.Specimen
+			0..0			LaboratoryObservationTopic.Focus
+			0..0			LaboratoryObservationTopic.Device
+			0..0			LaboratoryObservationTopic.Media
+							LaboratoryObservationTopic.EvaluationComponent
+			includes 0..1	TumorPrimaryDimensionSize
+			includes 0..1	TumorSecondaryDimensionSize
+			includes 0..1	SizeOfGrossTumorBed
+			0..0			LaboratoryObservationTopic.PanelMembers.Observation
+
+					Element:		TumorPrimaryDimensionSize
+					Description:	"The longest diameter of the primary tumor."
+					Based on:		EvaluationComponent
+					Concept:		SCT#399734001 "Primary tumor size (observable entity)"
+					Value:			Quantity with units UCUM#mm		
+					
+					Element:		TumorSecondaryDimensionSize
+					Description:	"The longest perpendicular diameter of the primary tumor."
+					Based on:		EvaluationComponent
+					Concept:		SCT#399738003 "Secondary tumor size (observable entity)"
+					Value:			Quantity with units UCUM#mm
+
+					Element:		SizeOfGrossTumorBed
+					Based on:		EvaluationComponent
+					Concept:		TBD			// mlt_to_rmb: needs new LOINC and SNOMED concept.
```

Comment:
@rutemartins to find appropriate code.  order of code selection:  LOINC (if observation), SNOMED, NCI Metathesaurus, local.

Reply:
@mlterryMitre to revisit whether `SizeOfGrossTumorBed` should be included for the purposes of staging. No changes at this point.